compiler-core/src/language_server/completer.rs

```diff
@@ -588,30 +605,57 @@ where
     }
 
     // Gets the position of the line with the first import statement in the file.
-    fn first_import_line_in_module(&'a self) -> Position {
-        let import_location = self
-            .module
-            .ast
-            .definitions
-            .iter()
-            .find_map(get_import)
-            .map_or(0, |i| i.location.start);
+    fn first_import_line_in_module(&'a self) -> (Position, bool) {
+        let first_import = self.module.ast.definitions.iter().find_map(get_import);
```

Comment:
Sorry- we always want imports to be added at the top, so could you make this check the first definition rather than searching for the first import? Thank you

Reply:
Hey, no worries. I used the first defnition. Please take a look. 

It's weird that the first definition is always "import" statement. So it didn't change the behavior 🤷 .

Reply:
Ok, apparently we can just use the `location.start` to find the 1st definition :)